images/runtime/php/8.2/base.bullseye.Dockerfile

```diff
@@ -1,148 +0,0 @@
-FROM oryxdevmcr.azurecr.io/private/oryx/php-8.2
```

Comment:
Curious why we are deleting base.*.Dockerfile?

Reply:
The comment from @pauld-msft above led to this removal: we previously added these auto-generated Dockerfiles to the `.gitignore`, but if the files were already a part of the repository when the changes to the `.gitignore` were made, Git doesn't remove the files. The command Paul mentioned will remove files that are already a part of the repository that match the `.gitignore` changes.

Reply:
Thanks for clarifying @cormacpayne!